.eslintrc.cjs

```diff
@@ -15,7 +15,7 @@ module.exports = {
 		'eqeqeq': ['error', 'always', { 'null': 'ignore' }],
 
 		// imports
-		'import/extensions': ['warn', 'never'],
+		'import/extensions': ['warn', 'always'], // "always" is most compatible with default vanilla ES Modules
```

Comment:
Having `.js` extensions, even if not to taste, is needed for the simplest ESM setup, and TypeScript officially recommends writing `.js` directly in TypeScript import statements for this purpose. So, we force it here so we don't mess up.

If we remove the `.js` extensions, the vanilla ESM example over at https://github.com/trusktr/prism-v2-test will break.